domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -290,7 +289,7 @@ public String toString() {
     spaceAppend(sb, offset.toString());
     String sql = sb.toString();
     if (log.isInfoEnabled()) {
-      log.info(getType() + " >>> SQL: " + getQuery() + " >>>" + sql);
+      log.info(String.format("%s >>> SQL: %s >>>%s", getType(), getQuery(), sql));
```

Comment:
```suggestion
      log.info("{} >>> SQL: {} >>>{}", getType(), getQuery(), sql));
```
String.format is slow and should be avoided for logging.